packages/core/src/interpreter.ts

```diff
@@ -135,14 +141,18 @@ export class Actor<
    */
   constructor(public logic: TLogic, options?: ActorOptions<TLogic>) {
     const resolvedOptions = {
-      ...defaultOptions,
+      ...Actor.defaults,
       ...options
-    };
+    } as ActorOptions<TLogic> & typeof Actor.defaults;
+
+    const { clock, logger, parent, id, systemId, inspect } = resolvedOptions;
 
-    const { clock, logger, parent, id, systemId } = resolvedOptions;
-    const self = this;
+    this.system = parent?.system ?? createSystem(this);
 
-    this.system = parent?.system ?? createSystem();
+    if (inspect) {
```

Comment:
I think there is no way atm to pass `inspect` when invoking or spawning children but perhaps better be safe than sorry here?
```suggestion
    if (inspect && !parent) {
```